### PR TITLE
docs: replace mixed type at `Encryption` class.

### PR DESCRIPTION
**Pull request by ping-yee**
**Description**
See #6310 

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Encryption/Encryption.php [hunk 1]
```diff
@@ -149,7 +149,7 @@ public static function createKey($length = 32)
      *
      * @param string $key Property name
      *
-     * @return mixed
+     * @return array|EncrypterInterface|string|null
```
Comment (kenjis) [inline system/Encryption/Encryption.php:152]: This method returns only 'key', 'digest', 'driver', 'drivers'. See `__isset()`.
```suggestion
     * @return array|string|null
```

File: system/Encryption/Handlers/BaseHandler.php [hunk 2]
```diff
@@ -61,7 +61,7 @@ protected static function substr($str, $start, $length = null)
      *
      * @param string $key Property name
      *
-     * @return mixed
+     * @return LoggerInterface|null
```
Comment (kenjis) [inline system/Encryption/Handlers/BaseHandler.php:64]: `$logger` is null. See #7579
`array|bool|int|string|null` ?

Comment (ping-yee) [reply system/Encryption/Handlers/BaseHandler.php]: Done.

File: system/Encryption/Encryption.php (hunk 1 above)
Comment (ping-yee) [reply system/Encryption/Encryption.php]: Done.

**Review by kenjis — APPROVED** [review]

**Pull request merged**